README.md

```diff
@@ -1,28 +1,87 @@
 # Modelz Python SDK
 
+[Docs](https://tensorchord.github.io/modelz-py/) | [Templates](https://cloud.modelz.ai/templates) | [ModelZ](https://modelz.ai) | [ModelZ Docs](https://docs.modelz.ai/)
+
+[ModelZ](https://modelz.ai) is an MLOps platform, you can deploy serverless instance for machine learning models by packed Docker image, such as Stable Diffusion, Dolly, ImageBind, and so on...
+
+`Deployment` is an instance of any ML service deployed at `ModelZ`, you could create one and then make `inference` to it. 
+
+`Templates` are preset Docker images for `deployment`, which is widely acknowledged used ML models, official templates are built and maintained by `ModelZ` developers. While it's available for you to define your own `template` and `deployment`.
+
+The python SDK is designed for CURD to your `deployments`, and sent request to them to make `inference`. It 's an alternative of ModelZ WebUI operation, which could be more friendly with CI/CD pipelines or at model development.
+
+## Install
+
 ```shell
 pip install modelz-py
 ```
 
-## CLI
+## CLI usage
+
+We support these functions now:
+
+- create/update/list/delete deployment
+- make inference to deployments
+- get metric information of any deployment
+
+This functions will be supported in the future:
+
+- build image and push to registry
+
+**See [CLI Docs](https://tensorchord.github.io/modelz-py/cli.html) for all usages.**
+
+## Example
+
+### Create and infer to ModelZ deployment at terminal
+
+#### Step 1: Create deployment
+First, you need to create a deployment at ModelZ platform. We pick `Stable Diffusion` image for this example.
+To get more predefined images, see our [templates](https://cloud.modelz.ai/templates).
+
```

Comment:
We can add a link to get ModelZ APIKey and User ID.

Reply:
Add help message for them.